### PR TITLE
feat: initial work on open graph

### PR DESCRIPTION
**Pull request by Primdahl26**
# Description

Please include a summary of the change and which issue is fixed.

Fixes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by AndreasPB — APPROVED** [review]
image



But @Pankai222 should probably look at the descriptions

File: frontend/src/routes/person/[id]/+page.svelte [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
   import { getMostPopularBackdropPath } from "$lib/utils"
   import TopCard from "$lib/components/details/top_card.svelte"
   import PersonMedia from "$lib/components/details/person_media.svelte"
+  import Head from "$lib/components/head.svelte"
   import type { Person } from "$lib/generated"
   import type { PageData } from "./$types"
 
   export let data: PageData
   const { person } = data
+  const posterUrl = PYTHON_API + `/image/?size=original&path=/${person.profile_path}`
```
Comment (AndreasPB) [inline frontend/src/routes/person/[id]/+page.svelte:12]: This is more or less `IMG_ORIGINAL`

File: frontend/src/routes/tv/[id]/+page.svelte [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
   import Seasons from "$lib/components/details/seasons.svelte"
   import Person from "$lib/components/details/person.svelte"
   import TopCard from "$lib/components/details/top_card.svelte"
   import Recommendations from "$lib/components/details/recommendations.svelte"
+  import Head from "$lib/components/head.svelte"
   import type { PageData } from "./$types"
 
   export let data: PageData
   const { tv } = data
+  const posterUrl = PYTHON_API + `/image/?size=original&path=/${tv.poster_path}`
```
Comment (AndreasPB) [inline frontend/src/routes/tv/[id]/+page.svelte:13]: `IMG_ORIGINAL`

File: frontend/src/routes/movie/[id]/+page.svelte [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
   import { currentCountry } from "$lib/stores/country.js"
   import Person from "$lib/components/details/person.svelte"
   import TopCard from "$lib/components/details/top_card.svelte"
   import Recommendations from "$lib/components/details/recommendations.svelte"
+  import Head from "$lib/components/head.svelte"
   import type { PageData } from "./$types"
 
   export let data: PageData
   const { movie } = data
+  const posterUrl = PYTHON_API + `/image/?size=original&path=/${movie.poster_path}`
```
Comment (AndreasPB) [inline frontend/src/routes/movie/[id]/+page.svelte:12]: `IMG_ORIGINAL`

File: frontend/src/lib/variables.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 export const PYTHON_API = import.meta.env.VITE_API_PATH
 export const GO_API = import.meta.env.VITE_GO_API_PATH
 
+export const PYTHON_STATIC = `${PYTHON_API}/static`
```
Comment (AndreasPB) [inline frontend/src/lib/variables.js:4]: I think we should follow the logic of not mentioning the language, so we can hot-swap the host(like the IMG-variables)
```suggestion
export const STATIC = `${PYTHON_API}/static`
```

File: frontend/src/routes/movie/[id]/+page.svelte (hunk 3 above)
Comment (Primdahl26) [reply frontend/src/routes/movie/[id]/+page.svelte]: Done

File: frontend/src/lib/variables.js (hunk 4 above)
Comment (Primdahl26) [reply frontend/src/lib/variables.js]: Done

File: frontend/src/routes/tv/[id]/+page.svelte (hunk 2 above)
Comment (Primdahl26) [reply frontend/src/routes/tv/[id]/+page.svelte]: Done

File: frontend/src/routes/person/[id]/+page.svelte (hunk 1 above)
Comment (Primdahl26) [reply frontend/src/routes/person/[id]/+page.svelte]: Done

**Pull request merged**